services/management/mysql.go

```diff
@@ -183,5 +185,7 @@ func (s *MySQLService) Add(ctx context.Context, req *managementpb.AddMySQLReques
 	}
 
 	s.state.RequestStateUpdate(ctx, req.PmmAgentId)
+	s.vc.SyncAndUpdate()
```

Comment:
maybe it's better to call `RequestSoftwareVersionsUpdate` for better consistency in names.

Reply:
renamed